test/util/fake_fixnum.dart

```diff
@@ -0,0 +1,5 @@
+library fixnum;
```

Comment:
It might be fragile.
We dropped requirement for library directives long ago.
So, developers are free to remove them - it does not change semantic of the language according to the specification. Ideally we should check for `package:fixnum` URI.

OTOH, we control this package, so we might decide to keep the library directive because it is used.

Reply:
How would I check the URI?

I'm happy to make the change; one downside though is that for testing we would then need a dependency on the fixnum package. I guess that would be okay if it's a dev_dependency?

Please let me know. Thanks :)

Reply:
I think doing this would require some kind of infrastructure to support packages in linter tests. Something like automatically adding all (or requested by the test?) packages in `test/package_mocks/` to `SourceFactory` of the test. You don't need to do this, leave it as is for now.

Reply:
Sounds good to me. Thanks.